src/main/java/com/uid2/admin/vertx/service/SharingService.java

```diff
@@ -232,13 +232,8 @@ private void handleListAllowedSites(RoutingContext rc) {
             return;
         }
 
-        JsonArray listedSites = new JsonArray();
-        Set<Integer> allowedSites = keyset.getAllowedSites();
-        if (allowedSites != null) {
-            allowedSites.stream().sorted().forEach((listedSiteId) -> listedSites.add(listedSiteId));
```

Comment:
Why remove the sorting and deduping? Those were there for a reason.

Reply:
I see, the sorting and deduping for hashing? I've put them back